src/Elastic.Apm.AspNetCore/ApmMiddleware.cs

```diff
@@ -265,5 +286,7 @@ private static string GetHttpVersion(string protocolString)
 					return protocolString.Replace("HTTP/", string.Empty);
 			}
 		}
+
+		private bool hasBody(HttpRequest request) => request.Body != null;

```

Comment:
Another a small nit: we use capital method names, like `HasBody`. - not that we don't like Java :)

Reply:
Java conventions die hard... I'm back to capital casing.